tests/Common/TracerTestTrait.php

```diff
@@ -258,6 +258,8 @@ private function parseRawDumpedTraces($rawTraces)
                     return;
                 }
 
+                $rawSpan["duration"] = (int)($rawSpan["duration"] / 1000);
+                $rawSpan["start"] = (int)($rawSpan["start"] / 1000);
```

Comment:
While this change does not impact lines 294 and 295 (since the `/ 1000` has been removed there), it impacts lines 278 and 281 that now receive micro instead of nano. I don't see any correction to that. Is it expected?

Reply:
That's right, this change had to be done _because_ they received nano. But micro is expected.

Reply:
Ok then, this is why it was not failing. Thanks for the explanation.